src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -37,6 +41,17 @@ case class EntityAttributeRecord(id: Long,
                                  listIndex: Option[Int],
                                  listLength: Option[Int]) extends AttributeRecord
 
+case class EntityAttributeTempRecord(id: Long,
+                                     entityId: Long,
+                                     name: String,
+                                     valueString: Option[String],
+                                     valueNumber: Option[Double],
+                                     valueBoolean: Option[Boolean],
+                                     valueEntityRef: Option[Long],
+                                     listIndex: Option[Int],
+                                     listLength: Option[Int],
+                                     transactionId: String) extends AttributeTempRecord
```

Comment:
brainstorming - is it worth adding `extends EntityAttributeRecord` here (and in the other "TempRecord" classes)? You'd still have to type out the properties, with `override`, to get proper equality - but inheritance would help enforce the contract that `TempRecord` classes are just like their corresponding `Record` classes, with the addition of a transactionId.